run-tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Only run tests in second-level directories that have been changed in the last commit.
+mapfile -t dirs < <( git diff --dirstat=files,0 HEAD~1 | sed 's/^[ 0-9.]\+% //g')
+declare -a tested_dirs=()
+for dir in "${dirs[@]}"
+do
+  second_level_dir=$(echo "$dir" | awk -F/ '{print FS $2}')
+  second_level_dir="${second_level_dir:1}"
+
+  if [ ! -z "$second_level_dir" -a "$second_level_dir" != " " ]; then
+    if [[ ! " ${tested_dirs[*]} " =~ " ${second_level_dir} " ]]; then
+      tested_dirs+=($second_level_dir)
+      python -m pytest -s $second_level_dir
```

Comment:
I think this needs to include the complete relative path to the second level directory?

Reply:
Good catch. I messed up the git history,  clean PR/branch is [here](https://github.com/explosion/projects/pull/114).